apps/dashr-svm/app.R

```diff
@@ -0,0 +1,510 @@
+                      "-webkit-user-select" = "none",
+                      "-ms-user-select" = "none"
+                    ),
+                    children = list(
+                      dccGraph(
+                        id = "graph-line-roc-curve",
+                        style = list(height = "40%"),
+                        figure = plot_ly() %>%
+                          layout(
+                            paper_bgcolor = "#272b38",
+                            plot_bgcolor = "#272b38",
+                            showgrid = FALSE,
+                            xaxis = list(
+                              zeroline = FALSE, 
+                              showline = FALSE, 
+                              showticklabels = FALSE, 
+                              showgrid = FALSE
+                            ),
+                            yaxis = list(
+                              zeroline = FALSE, 
+                              showline = FALSE, 
+                              showticklabels = FALSE, 
+                              showgrid = FALSE
+                            )
+                          ),
+                        config = list(displayModeBar = FALSE)
+                      ),
+                      dccGraph(
+                        id = "graph-pie-confusion-matrix",
+                        style = list(height = "55%", marginTop = "5%"),
+                        figure = plot_ly() %>%
+                          layout(
+                            paper_bgcolor = "#272b38",
+                            plot_bgcolor = "#272b38",
+                            showgrid = FALSE,
+                            xaxis = list(
+                              zeroline = FALSE, 
+                              showline = FALSE, 
+                              showticklabels = FALSE, 
+                              showgrid = FALSE
+                            ),
+                            yaxis = list(
+                              zeroline = FALSE, 
+                              showline = FALSE, 
+                              showticklabels = FALSE, 
+                              showgrid = FALSE
+                            )
+                          ),
+                        config = list(displayModeBar = FALSE)
+                      )
+                    )
+                  )
+                )
+              )
+            )
+          )
+        )
+      )
+    )
+  )
+)
+
+app$callback(
+  output("slider-svm-parameter-gamma-coef", "marks"),
+  list(input("slider-svm-parameter-gamma-power", "value")),
+  function(power){
+    s <- 10 ** power
+    as.list(
+      setNames(
+        lapply(seq(1, 10, 2) * s, round, 8), 
+        seq(1, 10, 2)
+      )
+    )
+  }
+)
+
+app$callback(
+  output("slider-svm-parameter-C-coef", "marks"),
+  list(input("slider-svm-parameter-C-power", "value")),
+  function(power){
+    s <- 10 ** power
+    as.list(
+      setNames(
+        lapply(seq(1, 10, 2) * s, round, 8), 
```

Comment:
`round(seq(1, 10, 2) * s, 8)`  is fine I think

Reply:
Definitely! Not sure why I did it like that :man_facepalming: